### PR TITLE
Add new event

### PR DESCRIPTION
**Pull request by nsakande**

### CODE REVIEW

**Review by ssettipalli — DISMISSED** [review]

**Comment by nsakande** [comment]
@ssettipalli this PR needs new approval since after update of develop branch project structure there were confllicts and i rebased.
Thanks in advance!

**Comment by nsakande** [comment]
@ssettipalli could you check this PR please?

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/QueuedEventDao.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public interface QueuedEventDao extends SqlObject {
+public interface    QueuedEventDao extends SqlObject {
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/QueuedEventDao.java]: There is an excessive amount of whitespace here.

File: pepper-apis/src/main/resources/db-changes/schema/add_new_event_type_DDP-7415.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="nika_sakandeldize" id="add-dsm-notification-event-type-for-stoolkit-07-02-22">
+        <insert tableName="dsm_notification_event_type">
+            <column name="dsm_notification_event_type_code">STOOL_SENT</column>
+        </insert>
+    </changeSet>
+</databaseChangeLog>
```
Comment (bskinner) [inline pepper-apis/src/main/resources/db-changes/schema/add_new_event_type_DDP-7415.xml]: It looks like this file got caught up in the directory restructure. This should be moved over to `pepper-apis/dss-database/src/main/resources/db-changes/schema/`

**Comment by nsakande** [comment]
@bskinner  Thanks for comments, Updated PR.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**